doc/changelog.d/20230209_110024_mjaskelioff_builtins_overview.rst

```diff
@@ -0,0 +1,4 @@
+Added
```

Comment:
A couple of things:
1. Do we want a changelog for this directory? I guess... maybe?
2. We're doing these in markdown. To control that we need a symlink to the `scriv.ini` file in the root, like the other ones have.

Reply:
I though it could be useful if some note gets deprecated.

Reply:
Should I keep it?

Reply:
deleted